client/src/partials/reports_proposed/result_account/result_account.js

```diff
@@ -0,0 +1,109 @@
+angular.module('bhima.controllers')
+.controller('configureResult', [
+  '$scope',
+  '$http',
+  '$routeParams',
+  '$translate',
+  '$sce',
+  'validate',
+  // Prototype document building module, requests document given configuration obejct
+  function ($scope, $http, $routeParams, $translate, $sce, validate) {
+
+    // Configuration objects optionally passed to /report/build - drives configuration UI
+    var session = $scope.session = {},
+        dependencies = {},
+        generatedDocumentPath = null,
+        serverUtilityPath = '/report/build/result_account';
+
+    var configuration = {
```

Comment:
If these configurations are shared between reports, it might be useful to make a `ReportConfigService` that has these values (and others) so that each developer does not have to re-write them.